tests/cypress/latest/support/functions.ts

```diff
@@ -529,3 +529,21 @@ Cypress.Commands.add('checkFilter', ({
     (testFilterTwo) ? cy.contains('test-filter-two').should('exist') : cy.contains('test-filter-two').should('not.exist');
     (shouldNotMatch) ? cy.contains('shouldnotmatch').should('exist') : cy.contains('shouldnotmatch').should('not.exist');
 });
+
+Cypress.Commands.add('checkLabelSize', ({
+  sizeToCheck}) => {
+    cy.clickNavMenu(["Dashboard"]);
+    cy.getBySel('button-create-registration-endpoint')
+      .click();
+    if (sizeToCheck == "name") {
+      cy.addMachInvLabel({labelName: 'labeltoolonggggggggggggggggggggggggggggggggggggggggggggggggggggg', labelValue: 'mylabelvalue', useHardwareLabels: false});
+    } else if (sizeToCheck == "value") {
+      cy.addMachInvLabel({labelName: 'mylabelname', labelValue: 'valuetoolonggggggggggggggggggggggggggggggggggggggggggggggggggggg', useHardwareLabels: false});
```

Comment:
Maybe a variable could have been useful for this "too long" value ;-) but that's fine.